designs/credentialManagement/profileFileSupport.md

```diff
@@ -0,0 +1,47 @@
+# Profile File Support
+
+## Abstract
+
+This document describes the process of supporting the AWS Profile files in the [Credential Management](credentialManagement.md) system
+
+## Shared Credentials File(s)
+
+AWS SDKs and tools are capable of sharing credentials by storing them in a common location. Also known as shared credential files, the **config** and 
+**credentials** files are detailed at [CLI documentation][CliConfigDocs]. We try to comply as close as possible with the CLI credential resolution behavior, 
+but only the keys related to credential management are supported in the toolkit. 
+
+Supported keys:
+* `aws_access_key_id`
+* `aws_secret_access_key`
+* `aws_session_token`
+* `source_profile`
+* `external_id`
+* `role_session_name`
+* `mfa_serial`
+* `credential_process`
+* `sso_*` - See [SSO Document](ssoSupport.md)
+
+### File Parsing
+Parsing and merging of the **config** and **credentials** files is out of scope of this document and is handled by the Java SDK. Please see [ProfileFile](https://github.com/aws/aws-sdk-java-v2/blob/master/core/profiles/src/main/java/software/amazon/awssdk/profiles/ProfileFile.java).
```

Comment:
What actions do we take when it fails though? I think that should be in this section

Reply:
Added